### ISSUE TITLE
S3 Upload (ManagedUpload) return object

### ISSUE THREAD

**Issue by alloyking** [aws/aws-sdk-js#1331]
When uploading a file as non-multipart I get the following response:
Buket : 'blah',
Etag : 'xxxxxx',
Key : "something/somefile.png",
Location : "https://some-url-address",
key : "somethings/somefile.png"

Why the multiple keys for "key".  One that is uppercase and one lowercase?  This isn't really all that awkward until you perform a Multipart upload which doesn't return "key"  just the uppercase version.

**Comment by chrisradek** [comment]
@alloyking 
The reason `key` appears twice is due to historical reasons. `Key` (uppercase) is the correct field to use (and what is documented). Under the hood, upload calls `putObject` or does a multi-part upload. Only one of those operations actually returns the `key` back in the response, so the SDK filled in the other one. Since we found out users were already relying on existing behavior, we documented just the one `Key`, but left both in to avoid breaking those users until we do a major version bump.
